test/NonSilo.Tests/Serialization/BuiltInSerializerTests.cs

```diff
@@ -129,6 +130,9 @@ public void InternalSerializableTypesHaveSerializers()
             Assert.True(
                 environment.SerializationManager.HasSerializer(grainReferenceType),
                 $"Should be able to serialize grain reference type {grainReferenceType}.");
+            Assert.True(
+                environment.SerializationManager.HasSerializer(typeof(PubSubGrainState)),
+                $"Should be able to serialize internal type {nameof(PubSubGrainState)}.");
```

Comment:
This is supposed to cover a representative sample of internal types. I included a diff in another comment which shows the types which are now covered by this PR but which were not previously covered.